test/scripts/openshift-ci/deploy.odh.sh

```
@@ -22,100 +22,65 @@ set -eu
-# It will only bind when a pod (the operator) actually consumes it
-echo "PVC created (will bind when consumed by operator pod)"
+  echo "Creating PVC for custom KServe manifests..."
+  oc apply -f "${SCRIPT_DIR}/odh-operator-custom-manifests/pvc.yaml"
+  echo "PVC created (will bind when consumed by operator pod)"
 
-# Patch CSV to mount custom manifests volume
-echo "Patching ODH operator CSV to mount custom manifests volume..."
-CSV=$(oc get csv -n ${ODH_OPERATOR_NAMESPACE} -o name | grep opendatahub-operator | head -n1 | cut -d/ -f2)
-echo "Found CSV: $CSV"
+  echo "Patching operator CSV to mount custom manifests volume..."
+  CSV=$(oc get csv -n ${ODH_OPERATOR_NAMESPACE} -o name | grep "${OPERATOR_NAME}" | head -n1 | cut -d/ -f2)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)